lib/scope.js

```diff
@@ -192,36 +212,47 @@ module.exports = {
     }
 
     let filename = name_parts.join(`-`);
+    let safe_name = safe_filename( filename, { replacement: `_` });
```

Comment:
If we're changing file names, I'd raise #563 again, which to fix is just deleting lines 210-212.  Especially if we're going to shorten the name strings anyway.

Reply:
I will say that the tags make it easier for me to find the file the scenario is in since, of course, I don't have the scenario descriptions memorized, but I do know that a tag of @story_tables means it's in the story tables test file. Maybe that's just because I know this library well right now, though. Maybe it's not that helpful to others?

Reply:
I don't have the scenario description memorized either, but I also don't remember enough of the existing tests (in any repo) to be able to recognize any of the tags too. And in that case, half of the tags are used on many other tests as well and get in the way of easily recognizing the one you need. I have to search for the description verbatim in the project, which is the same in the file name, unlike the tags, which have dashes added between them.